fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -3560,10 +3561,9 @@ private String serializeOperationOutcome(OperationOutcome oo) {
         }
     }
 
-    private synchronized CapabilityStatement getCapabilityStatement() throws Exception {
+    private synchronized CapabilityStatement getCapabilityStatement() throws FHIROperationException {
```

Comment:
After our earlier discussion, this is only called once? or does it synchronize across multiple requests, I could see this being slow unintentionally.

Reply:
I don't think this comment is really related to this change.  It was (and still is) called on each `metadata` request and thats what we have #734 for

Reply:
yeah ... I'm just wondering if we're injecting a pause with the synchronized when two threads are trying to generate it

Reply:
No need to resolve here